src/components/common/connections/Header.tsx

```diff
@@ -0,0 +1,80 @@
+import Breadcrumbs from '@mui/material/Breadcrumbs';
+import {
+  Link,
+  Typography,
+  ButtonGroup,
+  Stack,
+} from "@mui/material";
+import IconButton from "@mui/material/IconButton";
+import { vars } from "../../../theme/variables.ts";
+import {ArrowDown, ArrowRight, ArrowUp, HelpCircle} from "../../../icons";
+const { gray500} = vars
+
+const Header = ({showDetails, setShowDetails}: {showDetails: boolean, setShowDetails: (showDetails: boolean) => void}) => {
+  return (
+    <Stack
+      direction='row'
+      sx={{
+        alignItems:'center',
+        justifyContent:'space-between',
+        padding:'.75rem',
+        borderBottom:'1px solid #E3E5E8',
+        
+        '& .MuiSvgIcon-root': {
+          height: '1rem',
+          width: '1rem'
+        }
+      }}
+    >
+      <Stack
+        direction='row'
+        alignItems='center'
+        spacing='1rem'
+      >
+        {
+          showDetails &&  <ButtonGroup variant="outlined" sx={{
+            '& .MuiButtonBase-root': {
+              width: '2rem',
+              height: '2rem'
+            }
+          }}>
+                <IconButton>
+                    <ArrowUp />
+                </IconButton>
+                <IconButton sx={{ marginLeft: '.25rem' }}>
+                    <ArrowDown />
+                </IconButton>
+            </ButtonGroup>
+        }
+       
+        <Breadcrumbs
+          separator={<ArrowRight />}
+          aria-label="breadcrumb"
+        >
+          {showDetails ?
+            <Link underline="hover" onClick={() => setShowDetails(false)}>
+              Summary
+            </Link> : <Typography>
+                Summary
+              </Typography>
+          }
+          {
+            showDetails &&
+              <Typography>
+                  ilxtr:neuron-type-aacar-11
+              </Typography>
+          }
+        </Breadcrumbs>
+      </Stack>
+      
+      <Stack direction='row' alignItems='center' spacing='.5rem'>
+        <Typography variant='subtitle1' color={gray500}>
+          Displaying connection 1 of 5
```

Comment:
same here. A suggestion, I would create a js object that contains all the data we are expecting to render with the frontend. Doing so we are establishing a contract for the data structure we are expecting to receive in the components we are creating, and it avoid that someone has to rework the component to plug props instead of static text. Thanks!